analytics/src/main/java/com/gravatar/analytics/tracks/TracksTracker.kt

```diff
@@ -0,0 +1,25 @@
+package com.gravatar.analytics.tracks
+
+import com.automattic.android.tracks.TracksClient
+import com.gravatar.analytics.Event
+import com.gravatar.analytics.Tracker
+import java.util.UUID
+
+internal class TracksTracker(private val tracksClient: TracksClient) : Tracker() {
+    override var userId: String? = null
+    private val anonId: String = generateNewAnonID()
+
+    override fun trackEvent(event: Event) {
+        // We should add GRAVATAR userType when available.
+        tracksClient.track(event.name, userId ?: anonId, TracksClient.NosaraUserType.ANON)
```

Comment:
We should refactor the code when `NosaraUserType.GRAVATAR` is available.

Reply:
Do you know when we will have it?

Reply:
Not yet. If there are no changes in the PR today, I'll ask for the review and an estimated time for the new release.